app/src/main-process/squirrel-updater.ts

```diff
@@ -18,6 +18,9 @@ const exeName = Path.basename(process.execPath)
  */
 export function handleSquirrelEvent(eventName: string): Promise<void> | null {
   switch (eventName) {
+    case '--squirrel-firstrun':
+      return Promise.resolve()
```

Comment:
Ah yes, so I'm misunderstanding the intent here. So I'd like to "handle" this but do nothing - so we don't dump that message to the console. We don't really have an example of this, right? Do you care?